caffe2/operators/conv_pool_op_base.h

```diff
@@ -613,6 +606,13 @@ class ConvPoolOpBase : public Operator<Context> {
   }
 
  private:
+ inline void AllocateAndCopy = [&](
+        const vector<int>& vec, Tensor<Context>& tensor) {
```

Comment:
With a private member function you might just want to do 

inline void AllocateAndCopy(const vector<int>& vec, Tensor<Context>& tensor) {

right?